src/scripts/components/main/data-viewer/data-viewer.tsx

```diff
@@ -165,7 +168,7 @@ const DataViewer: FunctionComponent<Props> = ({
           action: () => setIsMainActive(false)
         })}
 
-      {showGlobeNavigation && <GlobeNavigation />}
+      {hideNavigation ? showGlobeNavigation : <GlobeNavigation />}
```

Comment:
This is super confusing! `showGlobeNavigation` is a boolean. Why do you want to render it here? Is it always false? when `hideNavigation` is true? Then you could just say `hideNavigation ? false : <GlobeNavigation />` or even `hideNavigation || <GlobeNavigation />`. 
Not really sure if I understand the logic here :)